terraform/modules/tf_stream_alert_cloudtrail/iam.tf

```diff
@@ -0,0 +1,45 @@
+// Role for CloudWatch events
```

Comment:
can this be extracted into its own module? it would be useful for the flow log module too